app/utils.py

```diff
@@ -0,0 +1,11 @@
+import re
+from flask_mail import Message
+
+def validate_email_format(email):
+    pattern = r'^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$'
+    return re.match(pattern, email) is not None
+
+def send_verification_email(email, mail):
+    msg = Message('Verification Email', sender='ahossain3324@gmail.com', recipients=[email])
```

Comment:
Try to avoid setting hard-coded values inside code. Rather, put a config file that will read values from the settings file. This way we will be able to make our code more dynamic. In case our email changes in the future or the API key gets expired, we don't need to check our code again. Rather, we will just update the settings file, and that's it.